tests/test_datamodel_api.py

```diff
@@ -66,6 +66,7 @@
 )
 
 
+@pytest.mark.skip(reason="https://github.com/ansys/pyfluent/issues/3910")
```

Comment:
@hpohekar I haven't seen this failing in other branches, I merged one yesterday. I'll first check the running status in the dev nightly.

Reply:
@mkundu1 Ok. right. I haven't checked nightly.

Reply:
@mkundu1 Yes, it's strange. Error logs shows that all these tests are not working properly hence created the issue.